### DISCUSSION TITLE
fromitive-blog/coding-interview/2023-07-11-hackerrank-1/

### DISCUSSION THREAD

**Discussion by giscus** [fromitive/fromitive-blog#10] [Q&A]
# fromitive-blog/coding-interview/2023-07-11-hackerrank-1/

HackerRank 문제 Hackerland Radio Transmitters 리뷰 (해결)

https://fromitive.github.io/fromitive-blog/coding-interview/2023-07-11-hackerrank-1/

**Comment by DDONONG** [comment]
시구널 보내
